website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

```diff
@@ -19,6 +19,15 @@ To set up a job to generate docs:
 
 4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.
 
+You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps. 
```

Comment:
I suggest swapping these so we can add that the "generate docs on run" procedure was outlined in the previous steps. I also added the word "check box" and "selecting" so it's slightly easier to associate with the rows in the table.
```suggestion
You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes when you add the command to the run steps compared to configuring a job selecting the **Generate docs on run** checkbox (shown in previous steps). 
```

Reply:
@mirnawong1 Do we have any docs that we can link to about adding to the list of commands in the job run steps? I couldn't find anything on a quick search but I must not have the right search terms.

Reply:
hey @runleonarun unfortunately we dont have anything specific, buuuuut @ernestoongaro and I are working on creating a document for this! see [here](https://www.notion.so/dbtlabs/Create-job-run-steps-docs-228d3e4c5a1f485592e1bd30e16f85ad)

Reply:
Oh right! That will be soooo helpful. I didn't realize we had nothing about them. 😍